### PR TITLE
Add a display panel beside personListPanel

### LINKED ISSUE #109: Add details panel next to the personListPanel

### PR DESCRIPTION
**Pull request by LemonDrew**
closes #109

### CODE REVIEW

**Review by oolongwater — COMMENTED** [review]
Perhaps information on the display panel when initialised?

File: src/test/java/seedu/address/logic/commands/CommandResultTest.java [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
```
Comment (SQ77) [inline src/test/java/seedu/address/logic/commands/CommandResultTest.java:20]:
```java
package seedu.address.logic.commands;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertNotEquals;
import static org.junit.jupiter.api.Assertions.assertNull;
import static org.junit.jupiter.api.Assertions.assertTrue;

import java.util.HashSet;
import java.util.Set;

import org.junit.jupiter.api.Test;

import seedu.address.model.person.Address;
import seedu.address.model.person.Email;
import seedu.address.model.person.Job;
import seedu.address.model.person.Name;
import seedu.address.model.person.Person;
import seedu.address.model.person.Phone;
import seedu.address.model.tag.Tag;
```

**Review by SQ77 — COMMENTED** [review]
Here is the import order for CommandResultTest.java

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 2]
```diff
@@ -19,6 +20,10 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    private boolean view = false;
```
Comment (SQ77) [inline src/main/java/seedu/address/logic/commands/CommandResult.java:23]: can this variable be named better as `isViewCommand`?

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="Address App" minWidth="800" minHeight="600" onCloseRequest="#handleExit">
```
Comment (SQ77) [inline src/main/resources/view/MainWindow.fxml:15]: Can the title be changed to "HiredFiredPro"?

File: src/test/java/seedu/address/logic/commands/CommandResultTest.java [hunk 4]
```diff
@@ -60,4 +73,44 @@ public void toStringMethod() {
...
+        CommandResult commandResult = new CommandResult("Feedback", true, null);
+        boolean result = commandResult.isView();
+        assertTrue(result, "Expected isView() to return true");
+    }
+
+    @Test
+    void isView_shouldReturnFalse_whenViewIsNotSet() {
+        CommandResult commandResult = new CommandResult("Feedback");
+        boolean result = commandResult.isView();
+        assertFalse(result, "Expected isView() to return true");
```
Comment (SQ77) [inline src/test/java/seedu/address/logic/commands/CommandResultTest.java:88]: Should this be "Expected isView() to return false"?

**Review by SQ77 — COMMENTED** [review]
some minor changes requested

**Review by bigismols — APPROVED** [review]
LGTM!

**Pull request merged**